cmd/gen-key/rsa.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"crypto"
+	"crypto/rsa"
+	"crypto/sha256"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"log"
+	"math/big"
+
+	"github.com/miekg/pkcs11"
+)
+
+func rsaArgs(label string, modLen, exp uint) ([]*pkcs11.Mechanism, []*pkcs11.Attribute, []*pkcs11.Attribute) {
+	expSlice := make([]byte, 8)
```

Comment:
Let's make this a ` expBytes [8]byte`, and turn it into a slice below with `expBytes[:]`.